aptos/nft_bridge/sources/complete_transfer.move

```diff
@@ -227,4 +227,4 @@ module nft_bridge::complete_transfer_test {
     }
 
     // TODO - failure test cases
-}
```

Comment:
RE commit conventions, is the idea here that commits like this are fine because you anticipate NFT bridge to be squashed into main?